### PR TITLE
Inject additional schools into pledge map

### PR DESCRIPTION
**Pull request by jeremydstone**
To make our CS pledge map appear less sparse early on, we are going to add our verified CSP teaching schools (390 total) to the map. For the moment we will do this by drawing from a "sidecar" CSV file and adding that data to the data we draw from the database before saving to the FusionTable. (In the future, we might just remove this data injection entirely; or if we keep it, we might decide to put it directly in the database as fake forms. We are going to start with this easily removable form of injection, since we are uncertain of the future and don't want to have to go delete fake forms amidst the real ones.)

### CODE REVIEW

File: bin/cron/cspschools.csv [hunk 1]
```diff
@@ -0,0 +1 @@
...
530723001077,480985,LINDBERGH SENIOR HIGH SCHOOL,RENTON,WA,98058,1,18
530780003344,480495,SELKIRK HIGH SCHOOL,IONE,WA,99139,2,6
530792001317,481166,SHOREWOOD HIGH SCHOOL,SHORELINE,WA,98133,1,29
530825001385,481258,FERRIS HIGH SCHOOL,SPOKANE,WA,99223,1,26
530825001386,481260,ROGERS HIGH SCHOOL,SPOKANE,WA,99207,1,12
530825001388,481265,LEWIS & CLARK HIGH SCHOOL,SPOKANE,WA,99204,1,21
530825001407,481282,SHADLE PARK HIGH SCHOOL,SPOKANE,WA,99205,1,11
530825003378,481283,ON TRACK ACADEMY,SPOKANE,WA,99207,1,10
530927003415,481493,VANCOUVER ITECH PREPARATORY,VANCOUVER,WA,98686,1,20
530969001648,480710,WEST VALLEY HIGH SCHOOL,SPOKANE,WA,99212,1,12
```
Comment (breville) [inline bin/cron/cspschools.csv]: could we get newlines in here easily? :)

File: bin/cron/update_pledge_map [hunk 2]
```diff
@@ -54,6 +54,21 @@ begin
       row = [form[:id], school[:name], school[:zip], type]
       csv << row
     end
+
+    if File.file?('cspschools.csv')
+      CSV.foreach('cspschools.csv', headers: true) do |input_row|
+        school_id = input_row['nces_id']
+
+        # Retrieve the ZIP for this school district from the dashboard's database.
+        school = DASHBOARD_DB[:schools].where(id: school_id).first
```
Comment (breville) [inline bin/cron/update_pledge_map:63]: Could _almost_ hoist this into a helper function, but no big deal either way.

File: bin/cron/cspschools.csv (hunk 1 above)
Comment (jeremydstone) [reply bin/cron/cspschools.csv]: Done. Word saved newlines as \r (MacOS standard). I changed it to \n (Linux standard) and that displays better in GitHub. (The CSV parser is happy with either.)

**Comment by jeremydstone** [comment]
(Merging before CircleCI run completes because this does not affect Circle.)

**Pull request merged**